modules/samples/sample-clients/apim-event-client/build.xml

```diff
@@ -0,0 +1,87 @@
+<!--
+  ~ Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ WSO2 Inc. licenses this file to you under the Apache License,
+  ~ Version 2.0 (the "License"); you may not use this file except
+  ~ in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+<project default="run">
+
+    <property name="carbon.home" value="../../../"/>
+    <property name="src.dir" value="src/main/java"/>
+    <property name="temp.dir" value="temp"/>
+    <property name="class.dir" value="${temp.dir}/classes"/>
+    <property name="main-class" value="org.wso2.carbon.sample.apimevent.Client"/>
+
+    <property name="protocol" value="thrift"/>
+    <property name="host" value="localhost"/>
+    <property name="port" value="7611"/>
+    <property name="username" value="admin"/>
+    <property name="password" value="admin"/>
+    <property name="numberOfEvents" value="100" />
+
+    <path id="javac.classpath">
+        <pathelement path="${class.dir}"/>
+        <fileset dir="${carbon.home}/wso2/lib/plugins">
+            <include name="commons-pool_*.jar"/>
+            <include name="com.google.gson_*.jar"/>
+            <include name="org.apache.commons.logging_*.jar"/>
+            <include name="libthrift_0.9.*.jar"/>
+            <include name="slf4j.log4j12_1.7.*.jar"/>
+            <include name="slf4j.api_*.jar"/>
+            <include name="org.yaml.snakeyaml_*.jar"/>
+            <include name="disruptor_*.jar"/>
+            <include name="org.wso2.carbon.config_2.1.*.jar"/>
+            <include name="org.wso2.carbon.secvault_5.0.*.jar"/>
+            <include name="org.wso2.carbon.utils_2.0.*.jar"/>
+            <include name="org.wso2.carbon.databridge.agent_6.0.*.jar"/>
+            <include name="org.wso2.carbon.databridge.commons.thrift_6.0.*.jar"/>
+            <include name="org.wso2.carbon.databridge.commons.binary_6.0.*.jar"/>
+            <include name="org.wso2.carbon.databridge.commons_6.0.*.jar"/>
+            <include name="org.ops4j.pax.logging.pax-logging-log4j2_*.jar"/>
+            <include name="org.ops4j.pax.logging.pax-logging-api_*.jar"/>
+        </fileset>
+
```

Comment:
we can remove this additional space.

Reply:
fixed in 1f69fe0038ac7ea895a7fae7e7714add8fc82281